tests/tests_app/utilities/packaging/test_lightning_utils.py

```diff
@@ -9,14 +9,22 @@
     _verify_lightning_version,
 )
 
-# TODO: this is very sensitive test, need to be done
+# TODO: Resolve this sensitive test.
```

Comment:
I would suggest either removing the test or using `pytest.skip` instead of commenting it

Reply:
Done !